aioesphomeapi/core.py

```diff
@@ -169,4 +172,7 @@ class ReadFailedAPIError(APIConnectionError):
     57: SirenCommandRequest,
     61: ListEntitiesButtonResponse,
     62: ButtonCommandRequest,
+    63: ListEntitiesLockResponse,
+    64: LockStateResponse,
+    65: LockCommandRequest,
 }
```

Comment:
58, 59 and 60 are already reserved for Lock as per #142 
If you change to those here and in `api.proto` then this PR can be merged.